src/mctpd.c

```diff
@@ -64,7 +64,7 @@ static mctp_eid_t eid_alloc_max = 0xfe;
 static size_t MAX_PEER_SIZE = 1000000;
 
 static const uint8_t RQDI_REQ = 1 << 7;
-static const uint8_t RQDI_RESP = 0x0;
```

Comment:
I'd like to keep the `RQDI_RESP` define here, for symmetry with `RQDI_REQ`. Just add the `_MASK` alongside.